server/Games/words.json

```diff
@@ -0,0 +1,8 @@
+{ 
```

Comment:
Same as the last comment. If we remove it from the back end, might just remove it from here. I'll see about integrating the rest.